lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -44,6 +44,14 @@ def get_ios_app_id_from_archive_plist(archive_path, plist_path)
         UI.shell_error!("can't extract GOOGLE_APP_ID") if identifier.empty?
         return identifier
       end
+
+      def blank?(value)
+        value.respond_to?(:empty) ? !!value.empty? : !value
```

Comment:
I think this needs to be `value.respond_to?(:empty?)` (note the question mark in the symbol). I tested this out in irb:

```
 ~  irb
irb(main):001:0> [].respond_to?(:empty?)
=> true
irb(main):002:0> [].respond_to?(:empty)
=> false
```

Reply:
Also you shouldn't have to use `!!` to coerce the result of `.empty?` to a boolean since AFAICT it only returns true or false: https://apidock.com/ruby/Array/empty%3F

Reply:
Good catch. I actually got this code from the rails source https://apidock.com/rails/Object/blank%3F. Added a comment specifying where it's from and also removed the coercion.